index.android.js

```diff
@@ -17,7 +17,16 @@
  */
 
 import { AppRegistry } from 'react-native';
+import Storybook from './storybook';
 
-import App from './app/index';
+// https://github.com/facebook/react-native/issues/15902
+// Symbol polyfills
+global.Symbol = require('core-js/es6/symbol');
```

Comment:
I'm reading the docs for core-js usage because I was confused between the `core-js/es6/..` imports vs `core-js/fn/..` imports.

Can't you just use `import 'core-js/es6/symbol';` and that should be enough? same with map, set and array find?

Reply:
I'll give that a go

Reply:
Yep, did the trick!